appservice/src/siteFiles.ts

```diff
@@ -47,24 +54,19 @@ export async function listFiles(context: IActionContext, site: ParsedSite, fileP
  * Overwrites or creates a file. The etag passed in may be `undefined` if the file is being created
  * Returns the latest etag of the updated file
  */
-export async function putFile(context: IActionContext, site: ParsedSite, data: string | ArrayBuffer, filePath: string, etag: string | undefined): Promise<string> {
+export async function putFile(context: IActionContext, site: ParsedSite, data: string | ArrayBuffer, url: string, etag: string | undefined): Promise<string> {
     const options: {} = etag ? { ['If-Match']: etag } : {};
     const kuduClient = await site.createClient(context);
-    const result: AzExtPipelineResponse = (await kuduClient.vfsPutItem(context, data, filePath, options));
+    const result: AzExtPipelineResponse = (await kuduClient.vfsPutItem(context, data, url, options));
     return <string>result.headers.get('etag');
 }
 
 /**
  * Kudu APIs don't work for Linux consumption function apps and ARM APIs don't seem to work for web apps. We'll just have to use both
  */
-async function getFsResponse(context: IActionContext, site: ParsedSite, filePath: string): Promise<AzExtPipelineResponse> {
+async function getFsResponse(context: IActionContext, site: ParsedSite, url: string): Promise<AzExtPipelineResponse> {
     try {
         if (site.isFunctionApp) {
-            const linuxHome: string = '/home';
-            if (site.isLinux && !filePath.startsWith(linuxHome)) {
```

Comment:
Oh yeah, are you still considering this special case?

Reply:
We aren't using the path anymore so this isn't needed. I'm assuming the href returned by the API is always correct. Do you know how to test this case?

Reply:
I verified it works with a Linux Function App. Not sure how else to test it.

Reply:
That works for me. I honestly don't remember the full context of why we had this in the first place.